### PR TITLE
Fix gpu selector handling

### PR DESCRIPTION
**Pull request by lugi0**
Signed-off-by: Luca Giorgi <lgiorgi@redhat.com>

### CODE REVIEW

**Review by mwaykole — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]
lgtm

File: tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot [hunk 1]
```diff
@@ -56,20 +56,21 @@ Select Container Size
...
+    Page Should Not Contain    All GPUs are currently in use, try again later.
+    Wait Until Page Contains Element    xpath://button[contains(@aria-labelledby, "gpu-numbers")]
     ...    error=GPU selector is not present in JupyterHub Spawner
 
 Set Number Of Required GPUs
     [Documentation]  Sets the gpu count based on the ${gpus} argument
     [Arguments]  ${gpus}
-    Click Element  xpath:${JUPYTERHUB_DROPDOWN_XPATH}\[2]
-    Click Element  xpath:${JUPYTERHUB_DROPDOWN_XPATH}\[2]/ul/li[.="${gpus}"]
+    Click Element  xpath://button[contains(@aria-labelledby, "gpu-numbers")]
```
Comment (bdattoma) [inline tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot:66]: can you create a variable for the xpath? like it was done with  `JUPYTERHUB_DROPDOWN_XPATH`

Comment (jgarciao) [reply tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot:66]: This will be addressed in another PR. @lugi0 ^

Comment (lugi0) [reply tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot:66]: #543

**Pull request merged**